lib/src/logic.dart

```diff
@@ -575,6 +575,74 @@ class Logic {
   /// Greater-than-or-equal-to.
   Logic operator >=(dynamic other) => GreaterThanOrEqual(this, other).out;
 
+  /// Shorthand for a [Conditional] which increments this by [incrVal]
+  ///
+  /// By default for a [Logic] var, if no [incrVal] is provided result is ++var
```

Comment:
Can't name a variable `var` in Dart

Would also be nice to add `'s around the "var" so that it's clear its a variable